### ISSUE TITLE
Merge failed in combine_html_json_pbp

### ISSUE THREAD

**Issue by jnederlo** [HarryShomer/Hockey-Scraper#14]
I noticed the merge failed for the EDM/NJD game id=`2019020049` in `combine_html_json_pbp()` in `game_scraper.py`.

I checked and the reason was because the `html_df.Seconds_Elapsed` column was represented as `dtype object` and not a `float`. 

I think adding this line after the try would fix the problem:
```html_df.Seconds_Elapsed.astype(float)```

I would make a pull request, but I don't know if it would cause other errors to go un-checked or something like that.

**Comment by HarryShomer** [comment]
Thanks for catching this! I'll try to take a look at it this weekend

**Comment by HarryShomer** [comment]
@jnederlo Ok I took a closer look at this. The game scraped properly but threw the following  warning (I'm guessing you can tell but below it should say rows and not columns).

![image](https://user-images.githubusercontent.com/25351167/66709356-29fa6680-ed30-11e9-90f8-ff1d01ca0dd1.png)

The issue is that the html has 2 more rows than the json. The type of the Seconds_Elapsed column isn't what is causing the issue here (I also checked and it was a float). 

Was this the error you were talking about or was an actual Exception being thrown? If so please include the full traceback with the version you're using.

**Comment by jnederlo** [comment]
@HarryShomer I saw that warning, but it still didn't scrape properly, for me it silently threw an error because of the try block. It was giving me the Seconds_Elapsed column as a type 'Object' and not a float. Perhaps the NHL fixed it. I went into the package to investigate this thoroughly at the time. The extra rows were DELP which were right above the PENL inputs, but your scraper merge handles that so that wasn't the issue, like I said it was the type of Seconds_Elapsed. Putting in the line I suggested fixed the problem for me and I don't think it would hurt to add it in for future cases.
